### ISSUE TITLE
Not using the location of the raw genotype file in pedigree QA

### ISSUE THREAD

**Issue by covaruber** [Breeding-Analytics/bioflow#27]
Could we make the pedQa functions work without assuming that we will have access to the original genotype file? Assuming this will not be a realistic scenario when deploying in the servers where files cannot be saved (e.g., shinyapps.io server). Could you please modify the functions to use directly the numerical code or converting back numeric code to letter for your functions (at least provisionally)? otherwise when a user saves their work and comes back that data is not available anymore. I hope that makes sense.

Cheers,

**Comment by RAngelaPG** [comment]
Hi Eduardo!

As I had told you before, in order not to lose real and valuable information when doing quality control, we need the original encoding of the heatmap file, but if this is not possible and we have to adapt it, I would ask you to please disable it for a while I solve it, this time it will take a considerable time, we are in harvest season and there are many analyzes to be carried out.

Best
Angela

**Comment by covaruber** [comment]
Yes, I remember the conversation, maybe at that moment I forgot that we will not be able to save the original final in the servers. I will leave the module active for now but it means that when they come back to a new session to continue their analysis the file will not be there anymore. But yes, whenever you have time I hope we can find a way. I will try to take a look next week and see if I can help :) Still useful to keep it active for now I think. Good luck with the harvesting season Angela :)

Cheers,
Eduardo
________________________________
De: Angela Pacheco ***@***.***>
Enviado: jueves, 25 de abril de 2024 11:16 p. m.
Para: Breeding-Analytics/bioflow ***@***.***>
Cc: Gio Covarrubias ***@***.***>; Author ***@***.***>
Asunto: Re: [Breeding-Analytics/bioflow] Not using the location of the raw genotype file in pedigree QA (Issue #27)


Hi Eduardo!

As I had told you before, in order not to lose real and valuable information when doing quality control, we need the original encoding of the heatmap file, but if this is not possible and we have to adapt it, I would ask you to please disable it for a while I solve it, this time it will take a considerable time, we are in harvest season and there are many analyzes to be carried out.

Best
Angela

**Comment by RAngelaPG** [comment]
Yes, got it, I'll figure it out, thanks! :)

**Comment by covaruber** [comment]
Hi Angela, can I close this ticket? I have the impression that you pushed several changes after this ticket. Feel free to close the ticket if this has been addressed. Thanks! 👍
